aws/resource_aws_vpc_peering_connection.go

```diff
@@ -69,6 +75,13 @@ func resourceAwsVPCPeeringCreate(d *schema.ResourceData, meta interface{}) error
 		createOpts.PeerOwnerId = aws.String(v.(string))
 	}
 
+	if v, ok := d.GetOk("peer_region"); ok {
+		if _, ok := d.GetOk("auto_accept"); ok {
+			return fmt.Errorf("peer_region cannot be set whilst auto_accept is true when creating a vpc peering connection")
+		}
+		createOpts.SetPeerRegion(v.(string))
```

Comment:
The convention has been to address the structure's field directly, i.e.
`createOpts.PeerRegion = aws.String(v.(string))`

Reply:
@ewbankkit Missed that, thanks!

About to push the change up, should I rebase and recomment the test results as well?